packages/vscode-extension/src/devices/IosSimulatorDevice.ts

```diff
@@ -346,16 +350,36 @@ export class IosSimulatorDevice extends DeviceBase {
     await Promise.all(matches.map(terminateApp));
   }
 
+  private getNativeLogsPath() {
+    const deviceSetLocation = getOrCreateDeviceSet(this.deviceUDID);
+    return path.join(deviceSetLocation, this.deviceUDID, "data", "radon_ide_ios_process.log");
+  }
+
+  mirrorNativeLogs(logFile: string) {
+    this.nativeLogsOutputChannel = window.createOutputChannel("Radon IDE (iOS Native Logs)", "log");
```

Comment:
I think I'd like "Simulator Logs" better than "Native Logs" – we have too many "native" terms in RN already

Reply:
Changed in: https://github.com/software-mansion/radon-ide/pull/742/commits/1d7e107217d8e645ab8ce51d70ed2ac6876713dd